### PR TITLE
feat(replays): add migration for replay_id in discover

### PR DESCRIPTION
**Pull request by JoshFerge**
split out from https://github.com/getsentry/snuba/pull/5787

### CODE REVIEW

File: snuba/migrations/group_loader.py
Comment (cmanallen) [inline snuba/migrations/group_loader.py:334]: Why were these removed?

**Review by cmanallen — CHANGES_REQUESTED** [review]

File: snuba/migrations/group_loader.py
Comment (cmanallen) [reply snuba/migrations/group_loader.py]: Oh lol my review was too slow.

Comment (JoshFerge) [reply snuba/migrations/group_loader.py]: whoops, fixed. merge issue

File: snuba/migrations/group_loader.py
Comment (JoshFerge) [reply snuba/migrations/group_loader.py]: fixed, sorry about that.

**Review by cmanallen — APPROVED** [review]
lgtm

**Review by evanh — APPROVED** [review]
My understanding is that this should not be merged until _after_ https://github.com/getsentry/snuba/pull/5788.

So I am approving, but please wait until that other PR is merged.

**Comment by JoshFerge** [comment]
am i good to merge this?

**Pull request merged**
